webnode/src/components/storage/index.js

```diff
@@ -12,7 +12,6 @@ import Datamap from "datamap-generator";
 import { TEST_ETH_ADDRESS } from "../../config";
 
 import TreasureTable from "./toolbox/TreasureTable";
-import ConsentOverlay from "../consent-overlay";
 
```

Comment:
Is the consent overlay now included in some other way?

Reply:
idk, but it was unused in the file, i was getting es-lint errors.

Reply:
i commented it out so it wouldn't load it twice (once from the scripts bundle and once from the react one).

two options:
1) keep the scripts bundle in dev mode and leave the `storage/index.js` file how it is here (without the ConsentOverlay)
2) remove the scripts bundle in dev mode, and add the ConsentOverlay back into this file / component

Reply:
i vote option two, i actually removed it from the dev bundle already. I would just need to add this back in.

Reply:
sounds good to me